bin/sdk/gutenberg.js

```diff
@@ -95,6 +97,13 @@ exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 						betaBlocks: presetBetaBlocks,
 					},
 				} ),
+			new webpack.BannerPlugin( {
+				banner: [
+					new Date().toUTCString(),
+					`Commit: ${ String( execSync( 'git rev-parse HEAD' ) ).trim() }`,
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/29303/commits/984f04d8e99596dd9b3730e4acaf44bad14edae9 with `git -C <path>`:

> Run as if git was started in `<path>` instead of the current working directory.

https://git-scm.com/docs/git#git--Cltpathgt